tests/utils.py

```diff
@@ -21,6 +23,20 @@
 from smdebug.exceptions import TensorUnavailableForStep
 
 
+class Timer(ContextDecorator):
+    def __enter__(self):
+        self.start_time = time.perf_counter()
+        return self
+
+    def __exit__(self, *exc):
+        self.stop_time = time.perf_counter()
+        self.time_taken = self.stop_time - self.start_time
+
+
+def is_running_in_codebuild():
```

Comment:
Add a brief docstring. Explain that this environment variable is only set in CodeBuild, hence it can be used to check whether this is being run in CodeBuild or not.